packages/lb-components/src/views/MainView/toolHeader/SwitchPattern/index.tsx

```diff
@@ -64,12 +76,7 @@ const SwitchPattern = ({ toolName }: IProps) => {
       >
         {t('DetectionMode')}
       </Button>
-      <Button
-        type={globalPattern === EPointCloudPattern.Segmentation ? 'primary' : undefined}
-        onClick={updateSegmentation}
-      >
-        {t('SegmentationMode')}
-      </Button>
+      {forbidSegmentation ? <Tooltip title={forbidText}>{segmentButton}</Tooltip> : segmentButton}
```

Comment:
why don't you just use 'forbidText' without 'forbidSegmentation' ? This allows you saving one param

Reply:
Good idea！